editions/tw5.com/tiddlers/variables/thisTiddler Variable (Examples).tid

```diff
@@ -0,0 +1,18 @@
+created: 20230304123606350
+modified: 20230304124149055
+tags: [[thisTiddler Variable]] [[Variable Examples]]
+title: thisTiddler Variable (Examples)
+type: text/vnd.tiddlywiki
+
+The first example shows how <<.def thisTiddler>> works whithin transcluded template. Here the [[SampleTemplate]] is a template for demonstration.
+
+<$macrocall $name=".example" n="1" eg="""{{||SampleTemplate}}"""/>
```

Comment:
@kookma, good examples. It is better to keep the `n="x"` values unique within a tiddler. Since the first two examples both have `n="1"`, the "Try it" button expands and collapse both examples at the same time.

Reply:
Sure, I correct it by submitting another PR